content/docs/about/kubeflow.md

```diff
@@ -11,6 +11,44 @@ recreate other services, but to provide a straightforward way to deploy
 best-of-breed open-source systems for ML to diverse infrastructures. Anywhere 
 you are running Kubernetes, you should be able to run Kubeflow.
 
+## Getting started with Kubeflow
+
+Follow the [getting-started guide](/docs/started/getting-started) to set up your
+environment.
+
+Then read the [documentation](/docs/) to learn about the features of Kubeflow, 
+including the following guides to Kubeflow components:
+
+* Kubeflow includes services for spawning and managing 
+  [Jupyter notebooks](/docs/notebooks/). [Project Jupyter](https://jupyter.org/) 
+  is a non-profit, open source project that supports interactive data science 
+  and scientific computing across many programming languages.
+
+* [Kubeflow Pipelines](/docs/pipelines/pipelines-overview/) is a platform for 
+  building, deploying, and managing multi-step ML workflows based on Docker 
+  containers.
+
+* Kubeflow offers a number of [components](/docs/components/) that you can use
+  to build your ML training, hyperparameter tuning, and serving workloads across
+  multiple platforms.
+
+## What is Kubeflow?
+
+Kubeflow is *the machine learning toolkit for Kubernetes*.
+
+To use Kubeflow, the basic workflow is:
+
+* Download and run the Kubeflow deployment binary.
+* Customize the resulting configuration files.
+* Run the specified scripts to deploy your containers to your specific
+  environment.
+
+You can adapt the configuration to choose the platforms and services that you 
+want to use for each stage of the ML workflow: data preparation, model training,
+prediction serving, and service management.
+
+You can choose to deploy your workloads locally or to a cloud environment.
```

Comment:
Could "deploying workloads locally" give an impression that it's on bare metal local server/laptop? If so we should clarify that it has to be a k8s deployment.

Reply:
Done